server/types/responses/userEventResponse.ts

```diff
@@ -0,0 +1,6 @@
+import { ObjectId } from "mongoose";
+import { EventResponse } from "./eventResponse";
+
+export interface UserEventResponse extends EventResponse {
+	userid: ObjectId;
```

Comment:
`UserEvents` has the same interface. Only difference is that it extends the base Event interface

Reply:
haha you snuck in your inheritance. OK fine.